code/datums/components/fantasy/suffixes.dm

```diff
@@ -0,0 +1,95 @@
+/datum/fantasy_affix/cosmetic_suffixes
+	placement = AFFIX_SUFFIX
+	alignment = AFFIX_GOOD | AFFIX_EVIL
+
+	var/list/goodSuffixes
+	var/list/badSuffixes
+
+/datum/fantasy_affix/cosmetic_suffixes/New()
+	goodSuffixes = list(
+		"dexterity",
+		"constitution",
+		"intelligence",
+		"wisdom", 
+		"charisma",
+		"the forest",
+		"the hills", 
+		"the plains", 
+		"the sea", 
+		"the sun", 
+		"the moon", 
+		"the void", 
+		"the world", 
+		"many secrets", 
+		"many tales", 
+		"many colors", 
```

Comment:
color change on each use/attack/click? ability to color stuff like spray paint on a hit?